tests/ten/ten_cor_014/run.py

```diff
@@ -18,10 +18,26 @@ def execute(self):
         tx_receipt1 = network.transact(self, web3_usr, storage.contract.functions.store(1), account_usr, storage.GAS_LIMIT)
         tx_receipt2 = network.transact(self, web3_usr, storage.contract.functions.store(2), account_usr, storage.GAS_LIMIT)
 
+        # create another user and have them send funds to our test user
+        pk2 = self.get_ephemeral_pk()
+        web3_usr2, account_usr2 = network.connect(self, private_key=pk2, check_funds=False)
+        self.distribute_native(account_usr2, network.ETH_ALLOC_EPHEMERAL)  # Fund the second user
+    
+        # second user sends funds to our test user
+        transfer_amount = web3_usr2.to_wei(0.001, 'ether')
+        tx_receipt3 = network.transact(self, web3_usr2, {
+            'to': account_usr.address,
+            'value': transfer_amount,
+            'gas': 21000,
+            'gasPrice': web3_usr2.eth.gas_price,
+            'chainId': web3_usr2.eth.chain_id
+        }, account_usr2, verbose=True, txstr='transfer to test user')
+
         # list the personal transactions for the user
         txs = self.scan_list_personal_transactions(url=network.connection_url(), address = account_usr.address, offset=0, size=5)
         tx_hashs = [x['blockHash'] for x in txs['Receipts']]
         self.assertTrue(txs != None, abortOnError=True, assertMessage='Returned list is None')
-        self.assertTrue(len(txs) == 2)
+        self.assertTrue(len(txs) == 3)
```

Comment:
if we are listing personal transactions for user account_usr, why would this increase to 3 i.e. include the transaction for account_usr2?

Reply:
because `account_usr` is a recipient on that new tx and we're now returning these as part of this query -https://github.com/ten-protocol/ten-internal/issues/4101